.github/workflows/size-limit.yml

```diff
@@ -17,9 +17,7 @@ jobs:
         with:
           node-version: 16
           cache: yarn
-      - run: yarn --frozen-lockfile
-      - run: yarn build
-      - uses: andresz1/size-limit-action@v1.5.1
+      - uses: andresz1/size-limit-action@v1.7.0
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
-          directory: polaris-react/
```

Comment:
Removing `directory: polaris-react` should fix the `size-limit` failures in [PR #5803](https://github.com/Shopify/polaris/pull/5803) as `yarn build` was only being executed in `polaris-react` where what we actually need is to run a build for `polaris-react` and all its dependencies.